foreshadow/transformers/internals/tfidf.py

```diff
@@ -51,9 +52,10 @@ def fit_transform(self, X, y=None):
 
         Args:
             X: iterable
+            y: labels
 
         Returns:
-            array-like: Transformed samples
+            (array-like) Transformed samples
```

Comment:
The Google style guide specifies a colon before the type specification, unless of course darglint was erroring in this case. (https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html)